Automattic/wp-calypso
# Add JITM modal template

#### Changes proposed in this Pull Request

![image of modal](https://user-images.githubusercontent.com/1883296/129565589-7644ac34-2aa4-46d0-b6bb-bef0782b9b3f.png)

This adds a new template to display a modal on the screen as specified by the jitm engine. It's based on the `Guide` component from WordPress.

#### Testing instructions

1. Apply D65534-code to your sandbox
2. Sandbox the API
3. View this PR locally or the link below
4. See an awesome modal
